core/src/error.rs

```diff
@@ -154,6 +151,10 @@ pub enum ExitError {
 	/// Other normal errors.
 	#[cfg_attr(feature = "with-codec", codec(index = 13))]
 	Other(Cow<'static, str>),
+
+	/// Invalid opcode during execution or starting byte is 0xef. See [EIP-3541](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3541.md).
+	#[cfg_attr(feature = "with-codec", codec(index = 15))]
+	InvalidCode(Opcode),
```

Comment:
The index is manually set in the macro attribute. Should I still move it back?

Reply:
Or do you mean the int value of the enum variant will change? Anyway, I've moved it back but it's a bit inconsistent, since the index values are unordered.